src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -1224,14 +1239,34 @@ internal ReadOnlySpan<byte> ValueSpan
             }
         }
 
-        // TODO make public https://github.com/dotnet/runtime/issues/33388
-        internal static bool DeepEquals(JsonElement left, JsonElement right)
+        /// <summary>
+        /// Compares the values of two <see cref="JsonElement"/> values for equality, including the values of all descendant elements.
+        /// </summary>
+        /// <param name="element1">The <see cref="JsonElement"/> to compare.</param>
+        /// <param name="element2">The <see cref="JsonElement"/> to compare.</param>
+        /// <returns><see langword="true"/> if the two values are equal; otherwise <see langword="false"/>.</returns>
+        /// <remarks>
+        /// Deep equality of two JSON values is defined as follows:
+        /// <list type="bullet">
+        /// <item>JSON values of different kind are not equal.</item>
+        /// <item>JSON constants <see langword="null"/>, <see langword="false"/> and <see langword="true"/> only equal themselves.</item>
```

Comment:
```suggestion
        /// <item>JSON constants <see langword="null"/>, <see langword="false"/>, and <see langword="true"/> only equal themselves.</item>
```
Go Oxford :smile:

Reply:
As a Cambridge grad I find that offensive 😆